services/openuserjs/openuserjs-version.service.js

```diff
@@ -25,7 +25,7 @@ export default class OpenUserJSVersion extends BaseOpenUserJSService {
       })
     }
     return renderVersionBadge({
-      version: data.UserScript.version[0].value,
+      version: data.UserScript.version.at(-1).value,
```

Comment:
> Are both spellings of license / licence possible to appear in the JSON?

What does https://github.com/OpenUserJS/OpenUserJS.org/blob/master/public/pegjs/blockUserScript.pegjs#L69-L93 say it will do when a key of that is found? i.e. don't make shields work more than it should and let us do what we do best for the meta script API. You can always try test scripts with values on production or even load up an instance in development. There's a reason why the Unit Test script exists too... utilize it to see how we do things